http/testdata/tls

```diff
@@ -0,0 +1 @@
+../../../cli/tests/tls
```

Comment:
This should be reverted; alongside whole `http/testdata/tls` directory

Reply:
Fixed it. Copy-paste never yielded good results haha. Thanks for pointing it out :)